packages/@aws-cdk/aws-apprunner/README.md

```diff
@@ -18,3 +26,80 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
 ```ts
 import apprunner = require('@aws-cdk/aws-apprunner');
 ```
+
+## Introduction
+
+AWS App Runner is a fully managed service that makes it easy for developers to quickly deploy containerized web applications and APIs, at scale and with no prior infrastructure experience required. Start with your source code or a container image. App Runner automatically builds and deploys the web application and load balances traffic with encryption. App Runner also scales up or down automatically to meet your traffic needs. With App Runner, rather than thinking about servers or scaling, you have more time to focus on your applications.
+
+## Service
+
+The `Service` construct allows you to create AWS App Runner services with `ECR Public`, `ECR` or `Github`. You need specify either `image` or `code` for different scenarios.
+
+## ECR Public
+
+To create a `Service` with ECR Public, define the `image` property with `ContainerImage.fromEcrPublic()`:
+
+```ts
+new Service(stack, 'Service', {
+  image: ContainerImage.fromEcrPublic('public.ecr.aws/aws-containers/hello-app-runner:latest'),
+  port: 80,
+});
+```
+
+## ECR
+
+To create a `Service` from an existing ECR repository, use `ContainerImage.fromEcrRepository()`:
+
+```ts
+// import the existing ECR repository by name
+const repo = ecr.Repository.fromRepositoryName(stack, 'ExistingRepository', 'existing-repo');
+new Service(stack, 'Service', {
+  image: ContainerImage.fromEcrRepository(repo),
+  port: 80,
+});
+```
+
+
+To create a `Service` from local docker image assets being built and pushed to Amazon ECR,
+use `ContainerImage.fromDockerImageAssets()`:
+
+```ts
+const imageAssets = new assets.DockerImageAsset(stack, 'ImageAssets', {
+  directory, // your code assets directory with Dockerfile
+});
+new Service(stack, 'Service', {
+  image: ContainerImage.fromDockerImageAssets(imageAssets),
+  port: 80,
+});
+```
+
+## Github
+
+To create a `Service` from the Github repository, you need to specify an existing App Runner Connection.
+
+See [Managing App Runner connections](https://docs.aws.amazon.com/apprunner/latest/dg/manage-connections.html) for more details.
+
+```ts
+new Service(stack, 'Service', {
+  connection: Connection.fromConnectionArn(connectionArn),
+  code: CodeRepository.fromGithubRepository({
+    repositoryUrl: 'https://github.com/aws-containers/hello-app-runner',
+    branch: 'main',
+    runtime: CodeRuntime.PYTHON_3,
+  }),
+});
+```
+
+## IAM Roles
+
+You are allowed to define `instanceRole` and `accessRole` for the `Service`.
+
+`instanceRole` - The IAM role that provides permissions to your App Runner service. These are permissions that
+your code needs when it calls any AWS APIs. 
+
+`accessRole` - The IAM role that grants the App Runner service access to a source repository. It's required for
+ECR image repositories (but not for ECR Public repositories). If not defined, a new access role will be generated
+when required.
```

Comment:
Ugh so App Runner decided to go with different names than the equivalent roles for ECS... Great! Not one bit confusing!

Reply:
Yes! Very clear now.